src/components/ui/__tests__/tooltip.enhanced.test.tsx

```
@@ -0,0 +1,853 @@
+            "data-side": side,
+            "data-align": align,
+            "data-open": isOpen,
+            onMouseEnter: handleMouseEnter,
+            onMouseLeave: handleMouseLeave,
+            style: { display: isOpen ? "block" : "none" },
+            ...props,
+          },
+          children,
+        );
+      },
+    );
```

ESLint

[error] Component definition is missing display name

(react/display-name)